packages/atomic/src/components/common/button.tsx

```diff
@@ -9,6 +9,7 @@ import {
 export interface ButtonProps {
   style: ButtonStyle;
   onClick?(event?: MouseEvent): void;
+  onKeyDown?(event?: KeyboardEvent): void;
```

Comment:
This is not used in the rest of your PR.

Reply:
It was from my other pr but I dont need it anymore